sky.json

```diff
@@ -0,0 +1,28 @@
+{
+        "response": [
+                {
+                        "maintainer": "Mourya Baruah",
+                        "oem": "Xiaomi",
+                        "device": "Redmi 12 5G",
+                        "filename": "crDroidAndroid-13.0-20231102-sky-v9.10.zip",
+                        "download": "https://sourceforge.net/projects/crdroid/files/sky/9.x/crDroidAndroid-13.0-20231102-sky-v9.10.zip/download",
+                        "timestamp": 1698926754,
+                        "md5": "254ceea809702fa2c83d221527c4184e",
+                        "sha256": "bd69b84b2c9a71e51e21236c64ba4d6553f8fe9a38a1e94f8465760b496a89a2",
+                        "size": 1005594040,
+                        "version": "9.10",
+                        "buildtype": "Beta",
```

Comment:
Maybe Weekly or Monthly as Beta would represent early stage development
But ofc up to you